Jenkinsfile

```diff
@@ -1,13 +1,10 @@
 buildPlugin(configurations: [
           // Sanity on the minimum required Jenkins version
-         [ platform: "linux", jdk: "8", jenkins: '2.190.3' ],
-
-          // Sanity on the recent Jenkins version
-         [ platform: "linux", jdk: "8", jenkins: null ],
+         [ platform: "linux", jdk: "8" ],
 
          // + Java 11
-         [ platform: "linux", jdk: "11", jenkins: null, javaLevel: "8"],
+         [ platform: "linux", jdk: "11"],
 
          // + Windows
-         [ platform: "windows", jdk: "11", jenkins: null, javaLevel: "8" ],
-])
\ No newline at end of file
+         [ platform: "windows", jdk: "11"],
```

Comment:
Note that you may prefer to do the Windows build on JDK 8, so that if you get a weird error you know it is because of Windows not because of Java 11. (treating Linux + Java 8 as the default configuration)

Reply:
Yeah, looks reasonable. Changed the configuration to "Windows + JDK 8"